upp-rds-provisioner/README.md

```diff
@@ -1,40 +1,44 @@
-UPP RDS resource
-===
+# UPP RDS
 
-RDS provisioning scripts
+The RDS provisioner will:
 
+* Create a RDS instance using the specified Cloudformation template
+* Create or update an appropriate CNAME record for the instance
+* Load the database with factset data using [upp-rds-loader](https://github.com/Financial-Times/upp-provisioners/tree/master/upp-rds-provisioner/loader) tool
 
-How To run it
-------
+The decommissioning process will:
+
+* Delete the cloudformation stack based on the CLUSTER tag; which deletes the resources created by the stack
+* Delete the instance CNAME record
 
-Setting up your environment is done by:
-* Make sure you have Python and [Virtualenv](https://virtualenv.pypa.io/en/stable/) installed
-* check out project
-* run
-```
-$ virtualenv  --no-site-packages -p python2 venv
-$ source venv/bin/activate
-$ pip install -r requirements.txt
-$ deactivate
-```
-Your environment is now setup to run Ansible using Virtualenv. For more details see http://docs.python-guide.org/en/latest/dev/virtualenvs/.
 
-The variables it takes are:
-* cluster - e.g. `pub-semantic`
-* environment_type - e.g `t` or `p`
-* cluster_sg - The security group the machines that will connect to the RDS instance are in
+## Building the Docker image
+The RDS provisioner can be built locally as a Docker image:
 
-To run the Ansible script you need to do the following:
-* Run `source venv/bin/activate` to activate the Ansible environment
-* Run `AWS_SECRET_ACCESS_KEY=je7MtGbClwBF/2Zp9Utk/h3yCo8nvbEXAMPLEKEY AWS_ACCESS_KEY_ID=AKIAI44QH8DHBEXAMPLE  ansible-playbook -vvv rdsserver.yml --extra-vars "cluster=pub-semantic environment_type=t cluster_sg=sg-10101"`
+`docker build -t coco/upp-rds-provisioner:latest .`
```

Comment:
For local Docker builds, lets tag images as `local` rather than `latest` - just makes it easier to differentiate what image you're running.

I'm going through and updating the other provisioner readmes so they do that as well :)